cylance_protect/plugin.spec.yaml

```diff
@@ -289,3 +401,41 @@ actions:
         description: Detailed information about the device lockdown
         type: lockdown_response
         required: true
+  search_threats:
+    title: Search Threats
+    description: Finds and displays detailed information about one or more threats
+    input:
+      threat_identifier:
+        title: Threat Identifier
+        description: The threat(s) to search for. The input should be an array of threat names, MD5, or SHA256 hashes
+        type: "[]string"
+        required: true
+        example: ["9de5069c5afe602b2ea0a04b66beb2c0", "02699626f388ed830012e5b787640e71c56d42d8", "Example-Threat-Name"]
+      score:
+        title: Score
+        description: Filter the search by the Cylance score assigned to the threat. Accepts an integer within the range [-1,1]
+        type: integer
+        required: false
+        example: -1
+    output:
+      threats:
+        title: Threats
+        description: Detailed information about threats found
+        type: "[]threat"
+        required: true
+  search_threat_agents:
+    title: Search Threat Agents
+    description: Finds and displays detailed information about threat and device details for a threat
+    input:
+      threat_identifier:
+        title: Threat Identifier
+        description: The threat to search for. The input should be a threat name, MD5, or SHA256 hash
+        type: string
+        required: true
+        example: 9de5069c5afe602b2ea0a04b66beb2c0
+    output:
+      threat_agents:
```

Comment:
If this is just returning the device information, can we change the object name to `agents`